src/Chains.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache 2
+pragma solidity ^0.8.13;
+
+// In the wormhole wire format, 0 indicates that a message is for any destination chain
+uint16 constant CHAIN_ID_UNSET = 0;
+uint16 constant CHAIN_ID_SOLANA = 1;
+uint16 constant CHAIN_ID_ETHEREUM = 2;
+uint16 constant CHAIN_ID_TERRA = 3;
+uint16 constant CHAIN_ID_BSC = 4;
+uint16 constant CHAIN_ID_POLYGON = 5;
+uint16 constant CHAIN_ID_AVALANCHE = 6;
+uint16 constant CHAIN_ID_OASIS = 7;
+uint16 constant CHAIN_ID_ALGORAND = 8;
+uint16 constant CHAIN_ID_AURORA = 9;
+uint16 constant CHAIN_ID_FANTOM = 10;
+uint16 constant CHAIN_ID_KARURA = 11;
+uint16 constant CHAIN_ID_ACALA = 12;
+uint16 constant CHAIN_ID_KLAYTN = 13;
+uint16 constant CHAIN_ID_CELO = 14;
+uint16 constant CHAIN_ID_NEAR = 15;
+uint16 constant CHAIN_ID_MOONBEAM = 16;
+uint16 constant CHAIN_ID_NEON = 17;
+uint16 constant CHAIN_ID_TERRA2 = 18;
+uint16 constant CHAIN_ID_INJECTIVE = 19;
+uint16 constant CHAIN_ID_OSMOSIS = 20;
+uint16 constant CHAIN_ID_SUI = 21;
+uint16 constant CHAIN_ID_APTOS = 22;
+uint16 constant CHAIN_ID_ARBITRUM = 23;
+uint16 constant CHAIN_ID_OPTIMISM = 24;
```

Comment:
gnosis?

Reply:
I copy-pasted from the go sdk `vaa/structs.go`, so it must be missing there too.

Will add here and there as well.